src/plugins/vis_type_timelion/public/helpers/panel_utils.ts

```diff
@@ -40,9 +38,27 @@ export interface Axis {
   tickLength: number;
   timezone: string;
   tickDecimals?: number;
-  tickFormatter: ((val: number) => string) | ((val: number, axis: Axis) => string);
+  tickFormatter: ((val: number) => string) | ((val: number, axis: any) => string);
```

Comment:
why `axis: Axis `was replaced to `axis: any`?

Reply:
Because it's interface also using in timelion app. But in timelion visualization tickFormatter should have second parameter as option. I have changed to 'any' avoiding type errors.